beacon-chain/types/block.go

```diff
@@ -271,6 +272,20 @@ func (b *Block) isAttestationValid(attestationIndex int, db beaconDB, aState *Ac
 		return false
 	}
 
+	// Retrieve block from db using justified_block_hash and verify its slot number is the same as attestation justified slot number.
```

Comment:
Line too long, please split for more readability

Reply:
+1